actin/operational/sync_actin_data_to_vm

```diff
@@ -9,7 +9,11 @@ if [[ "${curr_host}" == "prod-vm-operations" ]]; then
 fi
 
 info "Syncing ACTIN patient data to local VM"
+gsutil -m rsync -r $(locate_actin_clinical_data_feed_bucket) $(locate_actin_clinical_input_feed_sync_directory)
```

Comment:
do we actually need to sync the feed? it comes from the `/data/actin/clinical/input_feed`